config/rubocop.yml

```diff
@@ -55,6 +57,8 @@ Rails:
   Enabled: true
 Rails/Date:
   EnforcedStyle: strict
+Rails/EagerEvaluationLogMessage:
+  Enabled: false
```

Comment:
This prevents rubocop from fixing this: `Rails.logger.debug { "Downloaded #{url}" }`